### PR TITLE
[WIP] Add geocoding language parameter

### PR DESCRIPTION
**Pull request by danswick**
Adds a geocoding language parameter and tests.

edit: shoot. I missed your PR, @EnzoEghermanne. Happy to close here in favor of https://github.com/mapbox/mapbox-sdk-js/pull/173 if we can get tests on that PR. 

cc @hjudge

### CODE REVIEW

**Comment by EnzoEghermanne** [comment]
I closed my PR, as yours already includes these tests.

File: lib/services/geocoding.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ function roundTo(value, places) {
  * @param {Array} options.bbox a bounding box argument: this is
  * a bounding box given as an array in the format [minX, minY, maxX, maxY].
  * Search results will be limited to the bounding box.
+ * Specify the language to use for response text and, for forward geocoding, query result weighting. Options are IETF language tags comprised of a mandatory ISO 639-1 language code and optionally one or more IETF subtags for country or script. More than one value can also be specified, separated by commas.
```
Comment (tristen) [inline lib/services/geocoding.js]: Missing `@param {string} options.language` for this?

Comment (danswick) [inline lib/services/geocoding.js]: Thanks for catching that! Added in https://github.com/mapbox/mapbox-sdk-js/pull/185/commits/b607c5cae45b6f5463a4780f51f75602ef26c178.

**Comment by danswick** [comment]
Per chat with @tristen, the failing tests are occurring on the master branch as well, so we're going to merge here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@
 - **Feature** Add Tilesets API [#165](https://github.com/mapbox/mapbox-sdk-js/pull/165)
 - **Feature** Add Tokens API [#155](https://github.com/mapbox/mapbox-sdk-js/pull/155)
 - **Feature** Add response.nextPage() method for pagination of list resources [#159](https://github.com/mapbox/mapbox-sdk-js/pull/159)
+- **Feature** Add language parameter to geocoding API. [#174](https://github.com/mapbox/mapbox-sdk-js/issues/174)
```
Comment (andrewharvey) [inline CHANGELOG.md]: Shouldn't this go in the notes for "1.0.0-beta9" ?

**Pull request merged**